src/test/java/org/opensearch/performanceanalyzer/jvm/ThreadListTest.java

```diff
@@ -57,10 +52,10 @@ public void testNullThreadInfo() {
 
         ThreadList.parseAllThreadInfos(infos);
 
-        Map<String, AtomicInteger> counters = StatsCollector.instance().getCounters();
+        /*Map<String, AtomicInteger> counters = StatsCollector.instance().getCounters();
```

Comment:
Can we remove this if we are not using this anymore?

Reply:
Done, fixed this test.